README.md

```diff
@@ -5,48 +5,56 @@
-How an individual negotiates their equity-based compensation or when they choose to make
-certain decisions, such as exercise stock options, can have **major positive or negative**
-financial outcomes.
-If a company fails, an employee may get no value at all for what they own in a company, or
-may lose money they paid for the stock or for taxes.
-Even if a company succeeds, an employee may fall into
-[catastrophic tax pitfalls](#tax-dangers) because they didn’t properly understand tax
-consequences.
-
-Many people learn the basic ideas of from personal experience or from colleagues or
-helpful friends who have been through it before.
-But it is well understood primarily by tax attorneys, corporate lawyers, and other
-[professionals](#seeking-professional-advice). Many are daunted by legalities and tax
-planning, a challenge even more acute when faced with a short window in which to make a
-key decision, like accept a job offer or exercise stock options.
+to recent examples like [Facebook](https://en.wikipedia.org/wiki/Facebook). Stock options
+helped the first 3,000 employees of Facebook split roughly **$23 billion** at the time the
+company became public ([Financial Times](https://www.ft.com/content/d6599ae0-5738-11e1-869b-00144feabdc0)).
+
+But the ways equity can be granted as compensation—in the forms of
+[restricted stock, stock options](#how-equity-is-granted), and
+[restricted stock units](#how-equity-is-granted)—are notoriously complex.
+Equity compensation involves many high-stakes decisions for those who give and receive it,
+but best practices are shrouded in confounding terminology, changing laws, and differing
+approaches between companies.
+If you talk to enough employees and hiring managers, you’ll hear stories of how they or
+their colleagues met with the painful consequences of not learning enough up front.
+Though many people learn the basic ideas from personal experience or from colleagues or
+helpful friends who have been through it before, equity compensation is best understood by
+tax attorneys, corporate lawyers, and [other professionals](#seeking-professional-advice).
+Understanding tax planning and other legalities is daunting to many individuals, a
+challenge made even more acute when facing a short window in which to make a key decision,
+like accepting a [job offer](#offers-and-negotiations) or
+[exercising stock options](#stock-options).
+
+[What to negotiate](#negotiation-tips) and [when to exercise](#some-stock-option-scenarios) are
+among the many equity-based decisions individuals have to make, any or all of which can
+have major financial outcomes.
+Furthermore, because the value of employee equity is determined by the fate of the
+company, when a company does not succeed, an employee’s equity may be
+[worth nothing](#what-is-private-stock-worth?), and the [costs](#taxes-on-equity-compensation) of
+[exercising](#stock-options) may not be recouped.
+When a company does succeed, an employee may fall into
```

Comment:
I'd prefered some sort of transition here, like "Even when..." or perhaps an emphasis, since it's a remarkably surprising fact that you can be bankrupted while doing well on paper.